src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
```

Comment:
```suggestion
  person: { 
    name, 
    age, 
    sex, 
    isMarried, 
    partnerName,
  },
```

Reply:
Good remark, please place desctruct props in column, it will be more cleaner